404.md

```diff
@@ -5,4 +5,7 @@ sitemap: false
 permalink: /404.html
 ---
 
-Sorry! The requested page is not available. Please choose a page from the menu or [contact us](/en/contact).
+This probably means we could not convert the content<br/> 
```

Comment:
@phillip-hopper - it seemed be the only way to move to the next line.  A blank line actually gives two linefeeds which looks strange.